payments/verifier.go

```diff
@@ -0,0 +1,41 @@
+package payments
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"fmt"
+
+	"github.com/brave-intl/bat-go/utils/httpsignature"
+)
+
+// the list of payment verifiers, mapping to individuals in payments-ops
+var validVerifiers = map[string]bool{
```

Comment:
1. Is it possible to define this as a `const` rather than a `var` to prevent the value from being modified later?

2. Is the scope of this variable only modifiable within the scope of functions within this module or could other modules modify it?

Ideally we'd like to make this an unmodifiable constant with a short lived lifetime, so thinking this may make sense to define within the scope of the LookupVerifier function if that's the only place that it's actually used.

Reply:
Will add a comment to make sure this value is not modified in future commits.

Reply:
@husobee I pushed the key management policy for this directly onto this branch since I figured it was easiest to manage that document inside this codebase.

You can view it here: https://github.com/brave-intl/bat-go/pull/1384/commits/40b70af296407f17bf79ed4a75d09024c6ae372f